storj-exporter.py

```diff
@@ -112,6 +113,14 @@ def collect(self):
       if self.sat_data[sat]['storageDaily']:
         storj_sat_day_storage.add_metric(["atRestTotal", sat], self.sat_data[sat]['storageDaily'][-1]['atRestTotal'])
     
+    for satellite in self.data['satellites']:
+      value = str(satellite['disqualified'])
+      if value == "None":
+        value = 0
+      else:
+        value = 1
+      storj_sat_disqualified.add_metric([satellite['id']], value)
+
```

Comment:
See if you can use  `add_iterable_metrics` method in the existing for loop for satellites instead.

Also I don't think we need to be strict about the value returned. I prefer to keep exporter just translating values from api transparently where possible (unless it's breaking something).  In this case we can set anything that's not `0` to show as `disqualified` on Grafana side and let exporter just return api values. What do you think?

Reply:
Ok, I will rework it to use that method. I tried yesterday but was having variable recasting problems as well as not having a good metric to present the string value. Thinking about it for a bit more though I could use a gauge metric with multiple labels and have a label for satellite and another for disqualified. The PromQL query would then be on the satellite returning the disqualified label as exactly what the API is returning.
That's actually super doable, I will redo this work and implement that. Thanks for the feedback.